src/main/java/com/josdem/vetlog/command/UserCommand.java

```diff
@@ -38,10 +38,10 @@ public class UserCommand implements Command {
     @Size(min = 1, max = 100)
     private String lastname;
 
-    @Size(min = 1, max = 250)
-    private String phone;
+    @Size(min = 10, max = 10)
+    private String mobile;
 
```

Comment:
In the future, we want to have this format forced by UI: (123) 456-7890
From now on, we can go with only numbers.